src/nodes/model/conv/Conv2D.ts

```diff
@@ -10,33 +10,31 @@ export default class Conv2D extends Node {
     this.addInputInterface('Input');
     this.addOutputInterface('Output');
 
-    this.addOption('Filters', 'IntegerOption');
+    this.addOption('Filters', 'IntegerOption', 1);
```

Comment:
What happens if you don't pass in a default value? Are you stuck?

Reply:
So the integer option input just displays as blank, which I just thought looked a bit odd. Perhaps I should implement it inside of the option as well, or default to 0 or something.

Reply:
Do you think that would be better?

Reply:
If it is blank, can you ever get it to display a number? If not then I think it would be better to add a default value inside the option that way we can never get into that situation

Reply:
If it's blank, then pressing up displays the number 1. So it's not broken, it's basically just initialised to 0 without displaying it.

Reply:
Oh okay then I think it would be better to give undefined liked we spoke about offline but happy if you disagree